docs/specs/beacon-chain.md

```diff
@@ -1860,13 +1860,13 @@ Modified in altair to use `MIN_SLASHING_PENALTY_QUOTIENT_BELLATRIX` instead of `
 
 ## Genesis
 
-Before the Ethereum beacon chain genesis has been triggered, and for every Ethereum proof-of-work block, let `candidate_state = initialize_beacon_state_from_eth1(eth1_block_hash, eth1_timestamp, deposits)` where:
```

Comment:
These mentions are OK because they reference pre-merge Eth1 blocks, which _are_ proof-of-work blocks.

Reply:
Yes, but do we care about that paragraph today?

Reply:
Not really. Feel free to remove it.